### PR TITLE
Highlighting user actions in editor tour

### LINKED ISSUE #789: tour guide: bring clarity if something is actionable
### Describe the Bug

_No response_

### Steps to Reproduce the Problem

take the tour

### Expected Result

there are clear indications when you need to do something

### Actual Result

you carefully need to read the text that you understand that something needs to be done

### PermaplanT version

master

### Operating System

Linux

### Browser

Firefox

### Versions of other relevant software/extensions

_No response_

### Additional info/logs

_No response_

### PR DESCRIPTION
**Pull request by ThorbenStaufer**
Tour steps containing an action the user should perform now have following formatting:
- A colored border around the step.
- The text detailing the action is displayed semi-bold.
- The text detailing the action is displayed in color.
- The text detailing the action is displayed as a new paragraph.

Resolves #789.

## Basics

- [ ] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [ ] The PR is rebased with current master.
- [ ] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Comment by ThorbenStaufer** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

File: frontend/src/config/i18n/de/guidedTour.json [hunk 1]
```diff
@@ -24,20 +24,29 @@
         "plants_layer_intro": "Die Pflanzen-Ebene ist das Herzstück deiner Planung. Sie erlaubt es dir, Pflanzen auf deiner Karte zu setzen und unterstützt dich, das Gesamtwohl deines Garten dabei zu beachten.",
         "plants_layer_select": "Nutze das Ebenen-Panel und klicke jetzt auf die angezeigte Schaltfläche, um zur Pflanzen-Ebene zu wechseln und Zugriff auf deren Funktionen zu bekommen.",
```
Comment (markus2330) [inline frontend/src/config/i18n/de/guidedTour.json:25]:
```suggestion
        "plants_layer_select_text": "Nutze das Ebenen-Panel, um zur Pflanzen-Ebene zu wechseln und Zugriff auf deren Funktionen zu bekommen.",
        "plants_layer_select_action": "Klicke jetzt auf die angezeigte Schaltfläche rechts vom Pfeil.",
```

File: frontend/src/config/i18n/en/guidedTour.json [hunk 2]
```diff
@@ -24,20 +24,29 @@
         "plants_layer_intro": "The plants layer is the heart of your planning. It allows you to place plants on your map and supports you with remembering the overall wellbeing of your garden.",
         "plants_layer_select": "Use the layers panel and click now on the highlighted button to switch to the plants layer and get access to its features.",
```
Comment (markus2330) [inline frontend/src/config/i18n/en/guidedTour.json:25]:
```suggestion
        "plants_layer_select_text": "Use the layers panel to switch to the plants layer and get access to its features.",
        "plants_layer_select_action": "Click now on the button indicated by the arrow.",
```

File: frontend/src/config/i18n/de/guidedTour.json [hunk 3]
```diff
@@ -24,20 +24,29 @@
...
-        "plants_layer_first_planting_search": "Mit diesem Icon kannst du das Suchfeld einblenden lassen, um nach einer beliebigen Pflanze in der umfangreichen PermaplanT-Datenbank zu suchen. Klicke jetzt darauf, um deine erste Pflanze zu suchen.",
-        "plants_layer_first_planting_select": "Suche jetzt nach einer Tomate und klicke sie an, um in den Platzierungsmodus zu wechseln. Keine Sorge, die Aktionen werden im Laufe dieser Einführung rückgängig gemacht werden.",
+        "plants_layer_first_planting_search_text": "Mit diesem Icon kannst du das Suchfeld einblenden lassen, um nach einer beliebigen Pflanze in der umfangreichen PermaplanT-Datenbank zu suchen.",
+        "plants_layer_first_planting_search_action": "Klicke jetzt darauf, um deine erste Pflanze zu suchen.",
+        "plants_layer_first_planting_select_text": "Du wirst nun deine erste Pflanze suchen und platzieren. Keine Sorge, die Aktionen werden im Laufe dieser Einführung rückgängig gemacht werden.",
+        "plants_layer_first_planting_select_action": "Suche jetzt nach einer Tomate und klicke sie an, um in den Platzierungsmodus zu wechseln.",
```
Comment (markus2330) [inline frontend/src/config/i18n/de/guidedTour.json:30]: Put this text field higher (where the search text should be written). Please check if quotes work.

```suggestion
        "plants_layer_first_planting_select_action": "Tippe jetzt den Suchbegriff \"Tomate\" ein. Wähle dann per Mausklick die Tomate aus, um in den Platzierungsmodus zu wechseln.",

```

File: frontend/src/config/i18n/en/guidedTour.json [hunk 4]
```diff
@@ -24,20 +24,29 @@
...
         "plants_layer_right_toolbar": "After switching the active layer, the sidebars will automatically update to show its features. The sidebar of the plants layer lets you select a plant from your available seeds or through the search bar.",
-        "plants_layer_first_planting_search": "Use this icon to show the search bar to search for any plant in the extensive PermaplanT database. Click now on it to search for your first plant.",
-        "plants_layer_first_planting_select": "Search now for a tomato and click it to switch to placing mode. Don't worry, everything will be undone during this introduction.",
+        "plants_layer_first_planting_search_text": "Use this icon to show the search bar to search for any plant in the extensive PermaplanT database.",
+        "plants_layer_first_planting_search_action": "Click now on it to search for your first plant.",
+        "plants_layer_first_planting_select_text": "You will now search and place your first plant. Don't worry, everything will be undone during this introduction.",
+        "plants_layer_first_planting_select_action": "Search now for a tomato and click it to switch to placing mode.",
```
Comment (markus2330) [inline frontend/src/config/i18n/en/guidedTour.json:30]:
```suggestion
        "plants_layer_first_planting_select_action": "Now type the search term \"tomato\". Then click on the tomato to switch to placing mode.",
```

File: frontend/src/config/i18n/de/guidedTour.json [hunk 5]
```diff
@@ -24,20 +24,29 @@
...
-        "plants_layer_second_planting_search": "Um eine andere Pflanzenart zu verwenden, wähle diese wieder über die rechte Seitenleiste aus. Klicke erneut auf das Such-Icon, um die Such-Zeile zu öffnen.",
-        "plants_layer_second_planting_select": "Nun suchst du nach einer Kartoffel. Wenn du sie gefunden hast, klicke auf sie um den Platzierungsmodus mit dieser Pflanze zu starten.",
+        "plants_layer_change_date_text": "Jetzt wo deine erste Pflanze platziert ist kannst du die Zeitleiste nutzen, um deine nächsten Aktionen zu einem anderen Zeitpunkt stattfinden zu lassen.",
+        "plants_layer_change_date_action": "Klicke die Schaltfläche an und wähle ein Datum in der Zukunft.",
+        "plants_layer_second_planting_search_text": "Um eine andere Pflanzenart zu verwenden, wähle diese wieder über die rechte Seitenleiste aus.",
+        "plants_layer_second_planting_search_action": "Klicke erneut auf das Such-Icon, um die Such-Zeile zu öffnen.",
+        "plants_layer_second_planting_select_text": "Nun suchst du nach einer Kartoffel.",
```
Comment (markus2330) [inline frontend/src/config/i18n/de/guidedTour.json:36]: Put this text field higher (where the search text should be written).

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great work! A few improvements are needed.

**Review by markus2330 — APPROVED** [review]
Well done, we are very happy with the tour! :heart:

**Pull request merged**